app/deliver_grant_funding/routes/reports.py

```diff
@@ -1777,6 +1802,7 @@ def add_question_validation(grant_id: UUID, question_id: UUID) -> ResponseReturn
     methods=["GET", "POST"],
 )
 @has_deliver_grant_role(RoleEnum.ADMIN)
+@collection_is_editable()
```

Comment:
Currently what you can see in "read only" is very very limited (i.e you can't click around in nested groups or see any info on conditions or validations)

I wonder if this decorator might be able to receive an optional param to only restrict on mutate (POST) requests so that it could be used for pages that should still load and present a limited version of themselves but want confident it won't run if somehow the user sneaks in a edit request